.editorconfig

```diff
@@ -1,12 +0,0 @@
-root = true
```

Comment:
RIP lol

Reply:
We could probably do this in ReadMe proper as well?

Reply:
Yeah we probably should remove it there if it's still there. Prettier takes care of all this for us now.